src/charm.py

```diff
@@ -841,10 +841,13 @@ def _run_wp_cli(self, cmd, timeout=60, combine_stderr=False):
         except ops.pebble.ExecError as e:
             result = Result(e.exit_code, e.stdout, e.stderr)
         if combine_stderr:
-            logger.debug(f"Run wp-cli command: {cmd}\noutput: {result.stdout}")
+            logger.debug(f"Run wp-cli command: %s\noutput: %s", cmd, result.stdout)
```

Comment:
Can the f in front of the string be removed now?

Reply:
Fixed, thanks!